src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -1629,11 +1629,20 @@ public Extras<Externalizable> getExtras() {
         return extras;
     }
 
+    /**
+     * @deprecated use {@link FormEntryController#addFilterStrategy(FilterStrategy)} instead
```

Comment:
Since we're doing a major version bump anyway, what do you think about removing these right away? It's very unlikely there are existing consumers and if there are it's an easy change. We'd just want to make sure to track these breaking changes somewhere so we can document them.

Reply:
I'd thought about that, but we'd need to do significant work to move `EvaluationContext` ownership to `FormEntryController` (as that's what these methods affect). I gave doing a minimal change (where we still end up with a coupling between `FormDef` and `FormEntryController` but the `EvaluationContext` moves) a go, but it was pretty involved.

Reply:
Ah, ok, I thought it was an easier change that that. Sounds good.